v3/src/components/graph/adornments/univariate-measures/univariate-measure-adornment-simple-component.tsx

```diff
@@ -258,7 +269,7 @@ export const UnivariateMeasureAdornmentSimpleComponent = observer(
       if (measure) {
         addAdornmentElements(measure, valueObjRef.current, newLabelObj)
       }
-    }, [model, helper.instanceKey, labelRef, addAdornmentElements])
+    }, [addAdornmentElements, helper.instanceKey, model.isVisible, model?.measures])
```

Comment:
I think it would be sufficient to use:
```suggestion
    }, [addAdornmentElements, helper.instanceKey, model])
```
The value of `model` is captured by the function such that the function would not work if the `model` changed and the callback were not invalidated. Changes to the properties of the model don't invalidate the function in the same way, i.e. the callback behaves as intended after such changes.